storage/storage_image.go

```diff
@@ -360,6 +361,22 @@ func (s *storageImageDestination) HasThreadSafePutBlob() bool {
 	return true
 }
 
+// SupportsBlobLocks indicates whether the ImageDestination supports blob
+// locking.
+func (d *storageImageDestination) SupportsBlobLocks() bool {
+	return false
```

Comment:
Well :)

Maybe this function is not necessary at all, after all, if this went through unnoticed.

Reply:
Lol. No, that was stupid mistake on my side (see https://github.com/containers/skopeo/pull/619#issuecomment-475138951). This was the last optimization to the code, required some rebasing and I missed committing the last change to the PR -.- Apologies for the confusion.